### PR TITLE
Blank Canvas Blocks: Add default styles for List block

### PR DESCRIPTION
**Pull request by danieldudzic**
Part of #3413

### CODE REVIEW

File: blank-canvas-blocks/experimental-theme.json [hunk 1]
```diff
@@ -144,7 +144,17 @@
 					"color": "#ccc",
 					"thickness": "2px",
 					"width": "150px"
+				},
+				"list": {
+					"fontFamily": "var(--wp--custom--font-family--base)",
+					"padding": {
+						"top": 0,
```
Comment (jffng) [inline blank-canvas-blocks/experimental-theme.json]: If these values aren't used by the styles, should we be creating custom variables for them?

File: blank-canvas-blocks/experimental-theme.json [hunk 2]
```diff
@@ -144,7 +144,17 @@
...
+						"right": 0,
+						"bottom": 0,
+						"left": "2 * var(--wp--custom--padding--horizontal)"
```
Comment (jffng) [inline blank-canvas-blocks/experimental-theme.json]:
```suggestion
						"left": "calc( 2 * var(--wp--custom--padding--horizontal) )"
```

File: blank-canvas-blocks/sass/blocks/_list.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+ul,
+ol {
+	font-family: var(--wp--custom--list--fontFamily);
```
Comment (jffng) [inline blank-canvas-blocks/sass/blocks/_list.scss]: I think this would get transformed to `--wp--custom--list--font-family`

**Review by scruffian — APPROVED** [review]
LGTM

**Comment by danieldudzic** [comment]
Thanks for merging this, @scruffian!

**Pull request merged**